commit 203148b1ed6d8f4bad8030ef64f0bc4083309010
Author: fanquake <fanquake@gmail.com>
Date:   Sat Oct 4 01:00:25 2025 +0100

    static fixes

--- a/qtbase/src/gui/configure.cmake
+++ b/qtbase/src/gui/configure.cmake
@@ -599,6 +599,7 @@ qt_config_compile_test(xcb_syslibs
         XCB::XFIXES
         XCB::XKB
         XCB::XCB
+        X11::Xau
     CODE
 "// xkb.h is using a variable called 'explicit', which is a reserved keyword in C++
 #define explicit dont_use_cxx_explicit
